src/manage_sql_nvts.c

```diff
@@ -61,6 +61,40 @@ blank_control_chars (char *string)
     if (iscntrl (*string) && *string != '\n') *string = ' ';
 }
 
+
+/* NVT related global options */
+
+/**
+ * @brief File socket for OSP NVT update.  NULL to update via OTP.
+ */
+static gchar *osp_vt_update_socket = NULL;
+
+/**
+ * @brief Get the current file socket for OSP NVT update.
+ *
+ * @return The path of the file socket for OSP NVT update.
+ */
+const gchar *
+get_osp_vt_update_socket ()
```

Comment:
To keep in mind for the future: these functions don't contain SQL so should go in a manage_nvts.c file.  I have the idea that the upper layers (gvmd.c, gmpd.c, etc) shouldn't be importing *_sql*.c files, but only a single interface file like manage_tickets.h.

Reply:
I was also considering this but I'd prefer to consistently move all the non-SQL helper functions in a separate PR.

Reply:
Sounds good.